basetest.pm

```diff
@@ -399,15 +412,21 @@ sub save_test_result ($self) {
     return $result;
 }
 
+sub _increment_test_count ($self, $max = $bmwqemu::vars{MAX_TEST_STEPS} // 50) {
```

Comment:
I thought that should be 50000 and not 50? Or am I missing something?

Reply:
seems like a problem in rebasing happened here. I will create a fix

Reply:
fixed in https://github.com/os-autoinst/os-autoinst/pull/2323/

Reply:
Fixed by #2323